test/e2e/reconciler/scheduling/multi_placements_test.go

```diff
@@ -170,18 +170,6 @@ func TestMultiPlacement(t *testing.T) {
 			return false, fmt.Sprintf("Failed to get service: %v", err)
 		}
 
-		bindings, _ := kcpClusterClient.ApisV1alpha1().APIBindings().List(logicalcluster.WithCluster(ctx, userClusterName), metav1.ListOptions{
```

Comment:
Could you explain what is the link between the removal of these lines and this PR ?